src/scss/settings/_breakpoints.scss

```diff
@@ -0,0 +1,8 @@
+$grid-breakpoints: (
+  'xs': 320px,
+  'sm': 576px,
+  'md': 768px,
+  'lg': 992px,
+  'xl': 1024px,
+  '2xl': 1280px
+) !default;
```

Comment:
The breakpoints are a combination of bootstrap breakpoints and what we use now. If you feel they should not be there will remove. They were added because I wanted to have a map of our breakpoints. This should not limit us to add any extra breakpoints per component/page basis as necessary.

Reply:
@mcibique @matallo any more thoughts regarding this?

Reply:
Same position, confirm with them the breakpoints (given that this is greenfield and we might not need them all), and keep those as a reference to be imported.

Reply:
I confirmed with UX team will keep the ones I mention above and if we make any amends we should inform them.